### DISCUSSION TITLE
Multible Accounts - dealing with duplicated transactions from imports

### DISCUSSION THREAD

**Discussion by Bond246** [firefly-iii/firefly-iii#8203] [Firefly III]
Hello,

i'm using firefly since some months and my first step was to just use it for my main bank account. 
Transactions are imported via fints-importer for german bank accounts. 

So Transactions to my savings account at a different bank where handled like normal Expense accounts. No problem until now.
But now i decided also to import my savings account to have a better overview about all accounts. I also imported transactions via fints-import for this account. 

But now after importing transactions for the savings account some transactions seemed to be move from a normale transaction to a transfer. Others are normal deposits.
In general on the savings account it looks like that deposits are only deposits but withdrawals are imported as transfers.

I like that firefly knows what transactions are transfers from one account to another. But its bad when only some of this transactions are transfers and all the others not.

So questions is what to do now? How can i clean this up? How do i support firefly in recognizing transfers?

Thanks

**Comment by Bond246** [comment]
ok it seems to be a problem of importing deposits.
FinTS (maybe) know the IBAN of a withdrawal but not a deposit. 

So a withdrawal gets a transfer because he knows that the remote account is managed in firefly and from withdrawals it doesn't know the remote IBAN.

The question is how to deal with that?

**Reply by Bond246** [reply]
I've modified the title while i know the issue. So the questions is how to deal with transfers and duplicated deposits regarding to imported transactions?

**Comment by JC5** [comment]
If the transactions have something else in common, like a description or something, you could use rules to convert them. Otherwise, you\ll have to convert them manually.

**Reply by Bond246** [reply]
Thanks,
i think for now i will close this test. It is not so important to see the balance of the savings account and i don't know what the simples way would be to convert the transactions.
